app/src/main/java/ch/heigvd/pro/b04/android/Poll/PollAdapter.java

```diff
@@ -44,8 +46,10 @@ public PollAdapter(PollViewModel state, LifecycleOwner lifecycleOwner) {
     public long getItemId(int position) {
         if (position == 0)
             return HEADER_ID;
+        if (position == 1)
+            return INSTRUCTION_ID;
 
-        return questions.get(position - 1).getIdQuestion();
+        return questions.get(position - 2).getIdQuestion();
     }
```

Comment:
To be consistent with the `EmojiGridAdapter`, a switch statement might be used here.

Reply:
Done